src/pspm_check_python_modules.m

```diff
@@ -28,4 +29,8 @@
     catch ME
```

Comment:
Do we want this function to give a print message for users to understand whether the module can be loaded, or do we also want to allow this function to tell PsPM whether python package exists or can be loaded?